tests/baselines/reference/constEnumToStringWithComments.js

```diff
@@ -23,15 +23,15 @@ let c1 = Foo["C"].toString();
 
 
 //// [constEnumToStringWithComments.js]
-var x0 = 100 /* X */..toString();
-var x1 = 100 /* "X" */..toString();
+var x0 = 100 /* X */.toString();

```

Comment:
We should only ever emit `..` if the preceding text is an *integer literal*. We do not need to emit `..` if the preceding text is a decimal literal, whitespace, a comment, or anything else. `100/**/..toString()` is not legal JavaScript, so the previous outcome for this test was invalid.